cmd/simulateSam/simulateSam.go

```diff
@@ -94,20 +87,45 @@ func getReadsPerContig(ref []fasta.Fasta, numReads int) []int {
 	return readsPerContig
 }
 
+func usage() {
+	fmt.Print(
+		"simulateSam - Simulate alignments to a reference sequence\n\n" +
+			"Currently only generates illumina-style paired sequencing data" +
+			"Read Length : 150bp, Average Insert Size: 50bp" +
+			"Usage:\n" +
+			" simulateSam [options] ref.fasta out.sam/bam\n" +
+			"options:\n")
+	flag.PrintDefaults()
+}
+
 func main() {
-	outFile := flag.String("o", "stdout", "output file")
-	refFile := flag.String("r", "", "reference fasta file")
+	var expectedNumArgs int = 2
 	numReads := flag.Int("n", 100, "number of read pairs to generate")
-	bamOutput := flag.Bool("b", false, "output file as .bam instead of .sam")
-	seed := flag.Int64("seed", 1, "set the seed for the simulation")
+	setSeed := flag.Int64("setSeed", 1, "set the seed for the simulation")
+	readLength := flag.Int("readLength", 150, "Set the read length for each paired end.")
+	fragmentLength := flag.Int("fragmentLength", 400, "Set the average library fragment size.")
+	fragmentStdDev := flag.Float64("fragmentStdDev", 50, "Set the library fragment size standard deviation.")
```

Comment:
Are there thresholds for readLength, fragmentLength, and fragmntStdDev? Do you check and fatal if a threshold isn't met? I'm trying to think if there should be a limit on readLength, >0, <ref size?

I also need help understanding what "library fragment size" is because I thought it would be the same as readLength.
Actually, after reading an Illumina help page (https://support.illumina.com/bulletins/2020/12/how-short-inserts-affect-sequencing-performance.html), is it that we're assuming a fragmentation library prep (so we get a pool of library fragments of different sizes), and fragments become the "DNA Insert" with the proper structure for Illumina sequencing, but they're still usually longer than readLength (e.g. 400bp >150bp), so we do paired end sequencing?